payments/stripe/widgets.py

```diff
@@ -36,7 +37,10 @@ def render(self, name, value, attrs=None):
         del final_attrs['id']
         if value != '':
             # Only add the 'value' attribute if a value is non-empty.
-            final_attrs['value'] = force_text(self._format_value(value))
+            if django.VERSION[0] >= 2:
+                final_attrs['value'] = force_text(self.format_value(value))
```

Comment:
Its safe to use `self.format_value` in Django 1.11 which is last `1.x` that we are supporting, so version check is here unnecessary.

Reply:
Ok, I reverted to the version without version check.

Reply:
Didn't know that. So the original change was fine, excuse my mistake @RemyDCF :)